bigtable/client/row.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BIGTABLE_CLIENT_ROW_H_
```

Comment:
I think the new format for header guards is:

```cpp
#ifndef GOOGLE_CLOUD_CPP_BIGTABLE_CLIENT_ROW_H_
```

Is that correct, @coryan? Do you want this changed in this PR, or leave this for a future PR?

Reply:
I see the PR was merged, but presumably this was not updated.

Are we standardizing on `GOOGLE_CLOUD_CPP_*` header guards? Is there a check via Clang or another tool that enforces the consistency? If not, should there be?

Reply:
Sent #135 to fix the current problems, and filed #136 to automatically prevent similar problems in the future.